service/src/main/java/tds/session/web/endpoints/HealthController.java

```diff
@@ -0,0 +1,18 @@
+package tds.session.web.endpoints;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/session")
```

Comment:
Yes, it will be http://<domain + subdomain>/session/isAlive, and I agree we probably don't need it for the health endpoint.  However, I tend to think of the base request mapping defining the entity you're acting against by calling the endpoint.  We happen to be naming our domain and subdomain "session.sbac.org" in this instance but there isn't a guarantee that naming will happen at other deployments.  So it could be http://smarterbalance.com/session/endpoint.  Or at least that is my thinking on it.